### ISSUE TITLE
Extract result ( prediction ) from tflite android

### ISSUE THREAD

**Issue by abdou31** [tensorflow/tensorflow#30134]
I have converted a model with ( .ckpt files ) to .pb frozen model and I will convert this frozen model to tflite model to get it worked on Android device.
In effects, The model is trained using CNN facial landmarks project to get a prediction of eye region by drawing points ( landmarks ) on the contour of the eye like this:

![dsaa](https://user-images.githubusercontent.com/19480228/60110918-3d77c680-976d-11e9-8716-d6121d8c3469.PNG)

**System information**

+ **Tensorflow Version** : 1.13
+ **OS version** : Windows 10 x64
+ **Android studio version** :3.4
+ **Graphic Card**: Nvidia Geforce 840m

My question is how can I extract the points got from logits layer ( last layer )?
Note that I have successfully print the coordinates of points : 
    `
logits = tf.layers.dense(
        inputs=dense1,
        units=80,
        activation=None,
        use_bias=True,
        name="logits")
`
 `   predictions_dict = {
        "name": features['name'],
        "logits": logits  
`
` predictions = estimator.predict(input_fn=_predict_input_fn)
        for _, result in enumerate(predictions):
            img = cv2.imread(result['name'].decode('ASCII') + '.jpg')
            print(result['logits']) # print the landmarks
            marks = np.reshape(result['logits'], (-1, 2)) * IMG_WIDTH
            for mark in marks:
                cv2.circle(img, (int(mark[0]), int(
                    mark[1])), 1, (0, 255, 0), -1, cv2.LINE_AA)
            img = cv2.resize(img, (512, 512))
            cv2.imshow('result', img)`

**Comment by ravikyram** [comment]
Just to verify you have converted .pb to Lite model .Thanks!

**Comment by abdou31** [comment]
I have converted the file to tflite

**Comment by ymodak** [comment]
This question is better asked on [StackOverflow](http://stackoverflow.com/questions/tagged/tensorflow-tensorflow) since it is not a bug or feature request. There is also a larger community that reads questions there.
